cmd/reissueoperator.go

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2020-2020 The NATS Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/nats-io/nkeys"
+	"github.com/nats-io/nsc/cmd/store"
+	"github.com/spf13/cobra"
+)
+
+func createReIssueOperatorCmd() *cobra.Command {
+	var params reIssueOperator
+	cmd := &cobra.Command{
+		Use:          "operator",
+		Short:        "Re-issues operator with a new identity and re-signs affected accounts",
+		Example:      `nsc resign operator`,
```

Comment:
why not `reissue` - instead of `resign` - to me a `resign` command would re-sign all entities with the same signing key to update their issued date. - This will also make the help match the command.

Reply:
example was bad. the command is `nsc reissue operator`